pepper-apis/config/application.conf.ctmpl

```diff
@@ -44,8 +44,8 @@
     {{end}}
 	},
     "slack": {
-        "hook": "{{$slack.hook}}",
-        "channel": "{{$slack.channel}}",
+        "hook": "",
```

Comment:
If hook/channel values are empty DSS doesnt try to send alerts to Slack channels using the problematic SlackAppender code.